templates/devspace.yaml.tpl

```diff
@@ -51,6 +51,9 @@ vars:
   DEVENV_DEV_SKIP_PORTFORWARDING:
     source: env
     default: "false"
+  DEVENV_DEV_ONLY_FOWARD_DLV:
+    source: env
+    default: "false"
```

Comment:
thought: For some reason, I thought the current default was `true`.

Reply:
I believe the intent is that the devenv PR sets it to true, which seems reasonable to me.

Reply:
That is correct.